### DISCUSSION TITLE
Session Start Error

### DISCUSSION THREAD

**Discussion by Ihtiaree** [SafeExamBrowser/seb-win-refactoring#938] [General]
Initially I installed it and then it worked but it got stuck in user interface installation. then I tried to force the laptop to shut down. but after restarting it looks like this
Windows 11 Insider

![Screenshot 2024-07-17 145742](https://github.com/user-attachments/assets/4bc2b0fa-454d-45f4-98b2-8d531ca129ca)

[2024-07-17_14h57m28s_Client.log](https://github.com/user-attachments/files/16262240/2024-07-17_14h57m28s_Client.log)
[2024-07-17_14h57m28s_Runtime.log](https://github.com/user-attachments/files/16262242/2024-07-17_14h57m28s_Runtime.log)

**Comment by dbuechel** [comment]
You'll need to allow SEB to use the location services in the Windows settings, see #928.

**Reply by Ihtiaree** [reply]
thank you very much it worked, I had a headache for 3 hours because of this problem. It really works. Thank You :)
